test/ProjectTemplates/Microsoft.Agents.AI.Templates.IntegrationTests/Snapshots/aiagent-webapi.GitHubModels.verified/aiagent-webapi/Program.cs

```diff
@@ -1,22 +1,24 @@
 ﻿using System.ClientModel;
 using System.ComponentModel;
 using Microsoft.Agents.AI;
+using Microsoft.Agents.AI.DevUI;
 using Microsoft.Agents.AI.Hosting;
 using Microsoft.Agents.AI.Workflows;
 using Microsoft.Extensions.AI;
 using OpenAI;
+using OpenAI.Chat;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // You will need to set the token to your own value
 // You can do this using Visual Studio's "Manage User Secrets" UI, or on the command line:
 //   cd this-project-directory
 //   dotnet user-secrets set GitHubModels:Token YOUR-GITHUB-TOKEN
-var credential = new ApiKeyCredential(builder.Configuration["GitHubModels:Token"] ?? throw new InvalidOperationException("Missing configuration: GitHubModels:Token. See README for details."));
-var openAIOptions = new OpenAIClientOptions { Endpoint = new Uri("https://models.inference.ai.azure.com") };
-
-var chatClient = new OpenAIClient(credential, openAIOptions)
-    .GetChatClient("gpt-4o-mini").AsIChatClient();
+var chatClient = new ChatClient(
+        "gpt-4o-mini",
+        new ApiKeyCredential(builder.Configuration["GitHubModels:Token"] ?? throw new InvalidOperationException("Missing configuration: GitHubModels:Token.")),
```

Comment:
Should we use "GITHUB_TOKEN" for this config value? It seems like it's pretty common to use "GITHUB_TOKEN" for personal access tokens. For example, that's what the [Aspire GitHub Models Integration](https://learn.microsoft.com/dotnet/aspire/github/github-models-integration?tabs=dotnet-cli#default-api-key-parameter) looks for by default. And it's what GitHub Actions use for [authentication in workflows](https://docs.github.com/en/actions/tutorials/authenticate-with-github_token). I get that the current config value is with the AI Chat Web App template currently uses.

Reply:
I snapped to the same config property name we used in the `aichatweb` template, but I think you're right that we should be consistent with the other resources you referenced instead. I'll change this.

Reply:
Updated to `GITHUB_TOKEN` and `OPENAI_KEY`, while leaving `AzureOpenAI:Key` and `AzureOpenAI:Endpoint` since that keeps those 2 settings together. I only made this change for the new `aiagent-webapi` template, leaving `aichatweb` alone.

/cc @MackinnonBuck @davidortinau @jongalloway